_config.yml

```diff
@@ -1,4 +1,7 @@
 title: test
+exclude:
+  - _RIOT
+  - _tools
```

Comment:
I know it's not related to the PR, but should we also include the `README.md` and `Makefile` files?

Reply:
Include them to the `exclude` :-)

Reply:
Done

Reply:
I don't think so. We should exclude them

Reply:
So is it ok in the current state ?